### DISCUSSION TITLE
Set defaults for comment viewing? i.e. number and ordering

### DISCUSSION THREAD

**Discussion by durka** [ankitpokhrel/jira-cli#794] [Q&A] [answered]
Is there a way to display comments in reverse order (oldest first)?

Also, is there a way to set a default for `--comments` in .config.yml? Ideally, I would like to load all comments by default, but being able to set a high number like 10 or 100 by default would be good enough.

**Comment by ankitpokhrel** [comment]
It is not possible at the moment but I'm open to add this config (PRs welcome :)).

Also curious about the usecase for displaying comments in reverse order 🤔

**Comment by ankitpokhrel** [comment, marked as answer]
This is done with #853 and will be available in the next release. You can either use `--comments` flag with `issue list` cmd or set `num_comments` key to desired number in your config.
